Tests/UpstreamIntegrationTests/UpstreamIntegrationTests.swift

```diff
@@ -73,32 +69,35 @@ class UpstreamIntegrationTests: TestBase {
     // MARK: 1st launch scenarios
     func testSendEvent_withStandardExperienceEvent_receivesExpectedEventHandles() {
         // Setup
-
+        // Test constructs should always be valid
+        let standardNetworkRequest = NetworkRequest(urlString: "https://obumobile5.data.adobedc.net/ee/v1/interact", httpMethod: .post)!
         // Setting expectation allows for both:
         // 1. Validation that the network request was sent out
         // 2. Waiting on a response for the specific network request (with timeout)
-        networkService.setExpectationForNetworkRequest(url: "https://obumobile5.data.adobedc.net/ee/v1/interact", httpMethod: HttpMethod.post, expectedCount: 1)
+        networkService.setExpectationForNetworkRequest(networkRequest: standardNetworkRequest, expectedCount: 1)
 
-        // Test
         let experienceEvent = ExperienceEvent(xdm: ["xdmtest": "data"],
                                               data: ["data": ["test": "data"]])
+        // Test
         Edge.sendEvent(experienceEvent: experienceEvent)
 
         // Verify
         // MARK: Network response assertions
-        let networkRequest = NetworkRequest(urlString: "https://obumobile5.data.adobedc.net/ee/v1/interact", httpMethod: .post)!
-        let matchedResponsePost = networkService.getResponsesFor(networkRequest: networkRequest, timeout: 5)
+        let matchedResponsePost = networkService.getResponsesFor(networkRequest: standardNetworkRequest, timeout: 5)
         XCTAssertEqual(200, matchedResponsePost.first?.responseCode)
```

Comment:
- renaming standardNetworkRequest -> interactNetworkRequest
- renaming matchedResponsePost -> receivedResponse
- also assert on number of responses received